backend/src/core/config.py

```diff
@@ -12,9 +12,18 @@ class Settings(BaseSettings):
     APP_NAME: str
     APP_VERSION: str
 
-    SECRET_KEY: str
-    ACCESS_TOKEN_EXPIRE_MINUTES: int = 1440
+    UPLOADS_PATH: DirectoryPath
+    TEMPLATES_PATH: DirectoryPath
+
     ALGORITHM: str = "HS256"
+    SECRET_KEY: str
+    ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24 * 30  # 30 days
```

Comment:
you set these parameters in .env
```suggestion
    ACCESS_TOKEN_EXPIRE_MINUTES: int
```